numpy/f2py/tests/util.py

```diff
@@ -319,6 +319,7 @@ class F2PyTest(object):
     module = None
     module_name = None
 
+    @dec.knownfailureif(sys.platform=='win32', msg='Fails with MinGW64 Gfortran')
```

Comment:
Where would the best place be to put the comments? There are multiple tests all with the same cause.

Reply:
Maybe open an issue and just put the issue number.